### ISSUE TITLE
Did onnx-tf prepare function support 3D CNN? 

### ISSUE THREAD

**Issue by tomanick** [onnx/onnx-tensorflow#532]
I used SlowFast model of pytorch(3D CNN), and I wanted to convert it to Tensorflow.
I have converted to onnx model but I could not convert it to Tensorflow .pb file because of some bugs of onnx-tf.
The shape of the data was [batch_size, channels, depth(frames along the time axes), Width, Height]
I found that same conv and pooling parameters (ex: kernel size, padding, stride were the same) which could work in pytorch but onnx-tf got different dimention in depth. Therefore, I could not concat the outputs of slow pathway and fast pathway.

Did the onnx-tf prepare function support 3D CNN or the computing method of 3D Conv and 3D pooling of onnx-tf was different from pytorch?

**Python, ONNX, ONNX-TF, Tensorflow version**

This section can be obtained by running `get_version.py` from util folder.
 - Python version:  3.6
 - ONNX version:  1.5
 - ONNX-TF version: 1.5
 - Tensorflow version:  1.14

**Comment by LukasMosser** [comment]
Currently, there is no support for 3D Upsampling for example. 
It would be great to have this feature in the next onnx release to support many 3D CNN applications.
